dev/CommonStyles/TestUI/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,684 @@
+                    <ComboBoxItem Content="Item 1"/>
+                    <ComboBoxItem Content="Item 2"/>
+                    <ComboBoxItem Content="Item 3"/>
+                </ComboBox>
+            </Grid>
+            <Grid Grid.Row="6" Style="{StaticResource RightColumn}">
+                <ComboBox CornerRadius="6">
+                    <ComboBoxItem Content="Item 1"/>
+                    <ComboBoxItem Content="Item 2"/>
+                    <ComboBoxItem Content="Item 3"/>
+                    <ComboBoxItem Content="Item 4"/>
+                    <ComboBoxItem Content="Item 5"/>
+                    <ComboBoxItem Content="Item 6"/>
+                    <ComboBoxItem Content="Item 7"/>
+                    <ComboBoxItem Content="Item 8"/>
+                    <ComboBoxItem Content="Item 9"/>
+                    <ComboBoxItem Content="Item 10"/>
+                    <ComboBoxItem Content="Item 11"/>
+                    <ComboBoxItem Content="Item 12"/>
+                    <ComboBoxItem Content="Item 13"/>
+                    <ComboBoxItem Content="Item 14"/>
+                    <ComboBoxItem Content="Item 15"/>
+                    <ComboBoxItem Content="Item 16"/>
+                </ComboBox>
+            </Grid>
+
+            <TextBlock Grid.Row="7" Text="ComboBox (IsEditable)" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="7" Style="{StaticResource LeftColumn}">
+                <ComboBox IsEditable="True">
+                    <ComboBoxItem Content="Item 1"/>
+                    <ComboBoxItem Content="Item 2"/>
+                    <ComboBoxItem Content="Item 3"/>
+                </ComboBox>
+            </Grid>
+            <Grid Grid.Row="7" Style="{StaticResource RightColumn}">
+                <ComboBox IsEditable="True" CornerRadius="6">
+                    <ComboBoxItem Content="Item 1"/>
+                    <ComboBoxItem Content="Item 2"/>
+                    <ComboBoxItem Content="Item 3"/>
+                    <ComboBoxItem Content="Item 4"/>
+                    <ComboBoxItem Content="Item 5"/>
+                    <ComboBoxItem Content="Item 6"/>
+                    <ComboBoxItem Content="Item 7"/>
+                    <ComboBoxItem Content="Item 8"/>
+                    <ComboBoxItem Content="Item 9"/>
+                    <ComboBoxItem Content="Item 10"/>
+                    <ComboBoxItem Content="Item 11"/>
+                    <ComboBoxItem Content="Item 12"/>
+                    <ComboBoxItem Content="Item 13"/>
+                    <ComboBoxItem Content="Item 14"/>
+                    <ComboBoxItem Content="Item 15"/>
+                    <ComboBoxItem Content="Item 16"/>
+                </ComboBox>
+            </Grid>
+
+            <TextBlock Grid.Row="8" Text="AutoSuggestBox" Style="{StaticResource SideHeader}"/>
+            <Grid Grid.Row="8" Style="{StaticResource LeftColumn}">
+                <AutoSuggestBox ItemsSource="{x:Bind AutoSuggestSource}"/>
```

Comment:
I *think* you could define this in XAML, which would make things a bit easier to read without needing to go to code-behind to find out what this is.

Reply:
I've tried a couple of times to define an AutoSuggestBox data source in markup but haven't ever been able to get it to work. Do you have an example I can reference? Because I'd love to dump the code for this.

Reply:
Huh, I could've sworn that there was a way to define a list in XAML, but now that I'm looking it up it looks like that there isn't one, at least not that I can find.  Never mind then, my bad.